h/static/scripts/login-forms/components/test/LoginForm-test.js

```diff
@@ -46,18 +47,27 @@ describe('LoginForm', () => {
   it('renders the login form', () => {
     const { elements } = createWrapper();
     const {
-      form,
       csrfInput,
       usernameField,
       passwordField,
       forgotPasswordLink,
+      cancelButton,
     } = elements;
 
-    assert.equal(form.prop('action'), routes.login);
     assert.equal(csrfInput.prop('value'), fakeConfig.csrfToken);
     assert.equal(usernameField.prop('value'), '');
     assert.equal(passwordField.prop('value'), '');
     assert.equal(forgotPasswordLink.prop('href'), routes.forgotPassword);
+    assert.isFalse(cancelButton.exists());
+  });
+
+  it('renders the cancel button for oauth', () => {
```

Comment:
```suggestion
  it('renders the cancel button in the OAuth popup window', () => {
```

Clarify what "for oauth" means here.